public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -209,7 +209,12 @@ export class PanelChrome extends Component<Props, State> {
       });
     } else {
       // The panel should render on refresh as well if it doesn't have a query, like clock panel
-      this.onRender();
+      this.setState(
+        (prevState) => ({
+          data: { ...prevState.data, timeRange: this.timeSrv.timeRange() },
+        }),
+        this.onRender
```

Comment:
think we can skip this onRender as the above will trigger a  state update same as onRender